index.html

```diff
@@ -48,7 +44,7 @@ <h1 class="logo" href="#"><a>Ra<span class="title-split">mda.js</span></a></h1>
         </main>
         <footer>

```

Comment:
If u are using the `float` then should be applied `clearfix` for it or try to replace on flex properties

![image](https://user-images.githubusercontent.com/44003977/218645805-2c3855a9-0331-4f7a-83dc-4c944e382bf3.png)